projects/angular/src/data/datagrid/datagrid-page-size.ts

```diff
@@ -13,14 +13,15 @@ import { Page } from './providers/page';
   template: `
     <ng-content></ng-content>
     <div class="clr-select-wrapper">
-      <select [class.clr-page-size-select]="true" [(ngModel)]="page.size">
+      <select id="pageSizeOptionsId" [class.clr-page-size-select]="true" [(ngModel)]="page.size">
```

Comment:
I am pretty sure you mean `[id]="pageSizeOptionsId"`. Otherwise, this will set the `id` to literally "pageSizeOptionsId" which isn't helpful.

Reply:
Added the binding,